dvc/objects/checkout.py

```diff
@@ -225,6 +159,9 @@ def _checkout(
     if not diff:
         return
 
+    links = test_links(cache.cache_types, cache.fs, cache.fs_path, fs, fs_path)
+    link = Link(links)
```

Comment:
Now we just test links once and don't even call unncecessary `_confirm_link_type` on every file (even though it was cached, but still was calling things like `getsize`, which are expensive).